context/app/static/js/shared-styles/sections/SectionButtonRow/SectionButtonRow.jsx

```diff
@@ -14,17 +13,6 @@ function SectionButtonRow({ leftText, buttons, ...props }) {
   );
 }
 
-SectionButtonRow.propTypes = {
-  /**
-   Text to be displayed in the left most available space. Usually a BottomAlignedTypography component.
-  */
-  leftText: PropTypes.element.isRequired,
-  /**
-   Button(s) to be displayed in the right most available space.
-  */
-  buttons: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.element), PropTypes.element]).isRequired,
-};
-
```

Comment:
These comments are used by Storybook as descriptions, are these prop types causing warnings? I'd prefer to keep them.

Reply:
There was an proptypes warning in the log, but I'm not remembering where. Simplest just to revert this for now, and get the rest in.